src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -30,24 +38,58 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        $sprykDefinitions = $this->getFacade()->getSprykDefinitions();
-        $sprykDefinitions = $this->formatForTable($sprykDefinitions);
+        $spryk = current((array)$input->getArgument(static::ARGUMENT_SPRYK));
+        if ($spryk !== false) {
+            $sprykDefinition = $this->getFacade()->getSprykDefinition($spryk);
+            $tableRows = $this->formatOptionsTable($sprykDefinition['arguments']);
+            $output->writeln(
+                (new FormatterHelper())
+                    ->formatBlock(sprintf('List of all "%s" options:', $spryk), 'info')
+            );
+
+            $this->printTable($output, $tableRows, ['Option']);
+
+            $optionalSpryks = $this->getOptionalSpryks($sprykDefinition);
+            if ($optionalSpryks === []) {
+                $output->writeln(
+                    (new FormatterHelper())
+                        ->formatBlock('Optional Spryks:', 'info')
+                );
+
+                $this->printTable($output, $optionalSpryks, ['Spryk']);
+            }
+
+            return;
+        }
+
+        $sprykDefinition = $this->getFacade()->getSprykDefinitions();
+        $sprykDefinition = $this->formatSpryksTable($sprykDefinition);
 
         $output->writeln('List of all Spryk definitions:');
+        $this->printTable($output, $sprykDefinition, ['Spryk name', 'Description']);
+    }
 
-        $table = new Table($output);
-        $table
-            ->setHeaders(['Spryk name', 'Description'])
-            ->setRows($sprykDefinitions);
-        $table->render();
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @param array $tableContent
+     * @param array $header
+     *
+     * @return void
+     */
+    protected function printTable(OutputInterface $output, array $tableContent, array $header): void
```

Comment:
I would switch the order of `$tableContent` and `$header`

Reply:
Done.